### ISSUE TITLE
Structure of the <nav>

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#879]
**Problematic behavior**
The left column looks like a navigation, the items in this column should be in a `<nav>` element

The buttons in the left column should be structured in a `<ul><li>` list to allow screen reader users to know the number of elements present in the navigation.

**Possible Solution**
- Add a `role="navigation"` to the element `<nav>`
- Add a list `<ul><li>`

Example :

```
<ul>
<li><button>Tous les documents</button></li>
<li><button>Mes documents</button></li>
<li><button>Partagés avec moi</button></li>
</ul>
```
